docker-compose.yml

```diff
@@ -0,0 +1,49 @@
+version: '3'
+
+services:
+  app:
+    build:
+      context: .
+      dockerfile: "Dockerfile.api"
+    command: python3 ohp.py --start-api-server
+    environment:
+      - FLASK_ENV=development
```

Comment:
where did we use this?

Reply:
It sets the Flask environment as development so we don't this warning -
"WARNING: This is a development server. Do not use it in a production deployment."